src/silx/io/utils.py

```diff
@@ -879,7 +879,9 @@ def get_data(url):
 
     .. seealso:: :class:`silx.io.url.DataUrl`
 
-    :param Union[str,silx.io.url.DataUrl]: A data URL
+    :param url: A data URL
+    :param check_schemas: If true, the url with be checked with known schemas
+                          even is no schema was specified
```

Comment:
I feel, this makes it more complicated than it should.

Reply:
It's up to you.

Just in case it introduce problem.

Reply:
The new feature replaces `raise ValueError...` so it shouldn't be a problem.